seatunnel-examples/seatunnel-flink-examples/pom.xml

```diff
@@ -35,7 +35,7 @@
         <dependency>
             <groupId>org.apache.seatunnel</groupId>
             <artifactId>seatunnel-core-flink</artifactId>
-            <version>${parent.version}</version>
+            <version>${project.parent.version}</version>
```

Comment:
please revert this change, duplicate to https://github.com/apache/incubator-seatunnel/pull/994

Reply:
Thanks, revert.